worlds/tloz/Rules.py

```diff
@@ -58,22 +59,25 @@ def set_rules(tloz_world: "TLoZWorld"):
             add_rule(world.get_location(location.name, player),
                      lambda state: state.has("Stepladder", player))
 
+    # Level 4 Access
+    for location in tloz_world.levels[4].locations:
+        add_rule(world.get_location(location.name, player),
+                 lambda state: state.has("Raft", player) or state.has("Recorder", player))
```

Comment:
This can use `has_any` instead
```suggestion
                 lambda state: state.has_any(("Raft", "Recorder"), player))
```